src/main/java/com/openshift/restclient/model/IReplicationController.java

```diff
@@ -70,7 +71,7 @@
 	 * @param containerPorts  the container ports
 	 * @param volumes		 the set of emptyDir volumes to add to the config
 	 */
-	void addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
+	IContainer addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
```

Comment:
Does this really break the backwards compatibility if it adds a return parameter.  User's can still use it without breaking their code.  I'll have to see when it was introduced